index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    
+    <header>
+        <section class="menu_icon">
+                <div></div>
+                <div></div>
+                <div></div>
+        </section>
+    </header>   
+    
+    <nav class="main_section">
```

Comment:
Please the container of this `headline` section should be a `section` tag and not a `nav` tag as it represents a generic standalone section of your project 😄

Reply:
Great, let me do it!